### DISCUSSION TITLE
[Supply] Image upload is failing

### DISCUSSION THREAD

**Discussion by Thangaraj-M** [fastlane/fastlane#20354] [Q&A]
The supply action return the below error.

Google Api Error: Invalid request - Your icon does not apply the new icon specification. Please follow the instructions at https://developer.android.com/google-play/resources/icon-design-specifications#api. On June 17, this API will automatically apply the new specification

If we updated the store listing manually in play console and then if we try the supply action, it is success.

Is there any changes happened in Google API?

**Comment by mirkos93** [comment]
Hello, same problem here..

**Comment by SalvatoreAD** [comment]
Me too, same problem

**Comment by YaTree** [comment]
Same here.

**Comment by markmuniz** [comment]
Same issue here. I've noticed that this only occurs if no app icon exists on the store listing yet.

I've created an issue [here](https://github.com/fastlane/fastlane/issues/20359).

**Comment by xalikoutis** [comment]
same here

**Comment by sonjz** [comment]
same issue independent from fastlane
posted to their ruby sdk:
https://github.com/googleapis/google-api-ruby-client/issues/10035

**Comment by Thangaraj-M** [comment]
This issue resolved automatically now.

**Reply by sonjz** [reply]
confirmed on my end too, thanks!
